debezium-server/debezium-server-core/src/test/java/io/debezium/server/TestDatabase.java

```diff
@@ -6,16 +6,20 @@
 package io.debezium.server;
 
 import java.time.Duration;
+import java.util.Collections;
+import java.util.Map;
 
 import org.junit.jupiter.api.Assertions;
 import org.testcontainers.containers.FixedHostPortGenericContainer;
 import org.testcontainers.containers.GenericContainer;
 import org.testcontainers.containers.wait.strategy.Wait;
 
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+
 /**
  * @author Jiri Pechanec
  */
-public class TestDatabase {
+public class TestDatabase implements QuarkusTestResourceLifecycleManager {
```

Comment:
@gunnarmorling  Maybe we can move https://github.com/debezium/debezium-ui-poc/blob/master/backend/src/test/java/io/debezium/configserver/util/PostgresInfrastructureTestResourceLifecycleManager.java from the UI PoC to `debezium-testing-testcontainers` as "PostgresTestResourceLifecycleManager" instead? This is related to @ani-sha's current work, isn't it?